binding/vue/README.md

```diff
@@ -110,89 +110,195 @@ const porcupineModel = {
 }
 ```
 
-### Initialize Porcupine
+### Porcupine in Vue 2
+
+Use `usePorcupine` and `init` to initialize `Porcupine`:
+
+```vue
+
+<script lang='ts'>
+import Vue, { VueConstructor } from 'vue';
+import { BuiltInKeyword } from '@picovoice/porcupine-web';
+import { PorcupineVue, usePorcupine } from '@picovoice/porcupine-vue';
+
+// Use Vue.extend for JavaScript
+export default (Vue as VueConstructor<Vue & PorcupineVue>).extend({
+  data() {
+    const {
+      state,
+      init,
+      start,
+      stop,
+      release
+    } = usePorcupine();
+
+    init(
+      ${ACCESS_KEY},
+      [BuiltInKeyword.Porcupine],
+      porcupineModel
+    );
+    
+    return {
+      state,
+      start,
+      stop,
+      release
+    }
+  },
+  watch: {
+    "state.keywordDetection": function (keyword: PorcupineDetection) {
+      if (keyword !== null) {
+        console.log(keyword.label);
+      }
+    },
+    "state.isLoaded": function (isLoaded) {
+      console.log(isLoaded)
+    },
+    "state.isListening": function (isListening) {
+      console.log(isListening)
+    },
+    "state.error": function (error) {
+      console.error(error)
+    },
+  },
+  onBeforeDestroy() {
+    this.release();
+  },
+});
+</script>
+```
+
+In case of any errors, watch for `state.error` to check the error message, else watch `state.isLoaded` to check if `Porcupine` has loaded.
```

Comment:
```suggestion
In case of any errors, watch for `state.error` to check the error message, otherwise watch `state.isLoaded` to check if `Porcupine` has loaded.
```